include/ginkgo/core/matrix/csr.hpp

```diff
@@ -549,10 +598,18 @@ class Csr : public EnableLinOp<Csr<ValueType, IndexType>>,
     void convert_to(Csr<ValueType, IndexType> *result) const override
     {
         bool same_executor = this->get_executor() == result->get_executor();
-        EnableLinOp<Csr>::convert_to(result);
+        // NOTE: as soon as strategies are improved, this can be reverted
+        result->values_ = this->values_;
+        result->col_idxs_ = this->col_idxs_;
+        result->row_ptrs_ = this->row_ptrs_;
+        result->srow_ = this->srow_;
+        result->set_size(this->get_size());
```

Comment:
Using `EnableLinOp<Csr>::convert_to(result);` only gets the shared_ptr not the copy, right?
Or keeping `EnableLinOp<Csr>::convert_to(result);` is also fine because you create a new strategy later